core/domain/question_services.py

```diff
@@ -320,6 +406,15 @@ def update_skill_ids_of_questions(
     question_models.QuestionSkillLinkModel.put_multi_question_skill_links(
         new_question_skill_link_models)
 
+    questions = get_questions_by_ids(list(question_ids))
+    for question in questions:
+        new_linked_skill_ids = copy.deepcopy(question.linked_skill_ids)
+        new_linked_skill_ids.remove(curr_skill_id)
+        new_linked_skill_ids.append(new_skill_id)
+        _update_linked_skill_ids_of_question(
```

Comment:
Should this be made into a put_multi()-type operation?

Reply:
So I was trying to figure that..I was hesitant to do a delete_multi/put_multi for question objects..is that safe?

Reply:
Done